src/njoy21/input/LEAPR/Card1/Nout/test/Nout.test.cpp

```diff
@@ -0,0 +1,33 @@
+#define CATCH_CONFIG_MAIN
```

Comment:
No, I'm referring to the entire file. If we are going to use the `common::Nout` instead, we don't have to have this test file.